test/unit/test_configcommon.py

```diff
@@ -124,3 +125,38 @@ def test_clean_dynamic_variables(autosubmit_config: Callable) -> None:
 
     assert len(as_conf.dynamic_variables) == 1
     assert 'jaspion_eats' in as_conf.dynamic_variables
+
+
+def test_yaml_deprecation_warning(tmp_path, autosubmit_config: Callable):
+    """Test that the conversion from YAML to INI works as expected, without warnings.
+
+    +    Creates a dummy AS3 INI file, calls ``AutosubmitConfig.ini_to_yaml``, and
+    +    verifies that the YAML files exists and is not empty, and a backup file was
+    +    created. All without warnings being raised (i.e. they were suppressed).
+    +    """
+
+    as_conf: AutosubmitConfig = autosubmit_config(expid='a000', experiment_data={})
+    ini_file = tmp_path / 'a000_jobs.ini'
+    with open(ini_file, 'w+') as f:
+        f.write(dedent('''\
+            [LOCAL_SETUP]
+            FILE = LOCAL_SETUP.sh
+            PLATFORM = LOCAL
+            '''))
+        f.flush()
+    as_conf.ini_to_yaml(root_dir=tmp_path, ini_file=str(ini_file))
+
+    print(f'tmp_path: {tmp_path}')
+    with open(tmp_path / 'a000_jobs.ini_AS_v3_backup', 'r') as backup, open(tmp_path / 'a000_jobs.yml', 'r') as newYaml:
+        lines_backup = backup.readlines()
+        lines_newYaml = newYaml.readlines()
```

Comment:
Camel case is not used in Python. So never `newYaml`, only snake case like `new_yaml` is used.

Reply:
You may want to configure your editor or IDE so issues like this are automatically flagged, before you push to branches. Here's what PyCharm looks like when I open your code.

![image](https://github.com/user-attachments/assets/905e39dd-f785-4b2e-a8e3-524a83d722f7)

There are light wiggly lines under some parts of the code, that also appear on the right sidebar, and as a summary at the top right of my IDE. It shows 2 warnings for this file (the `False` at the end, and this camel case).